### PR TITLE
verify private users in get_links_for_username; small fixes

### PR DESCRIPTION
**Pull request by sionking**
I also think we shall maintain names: 'person' for target user and 'username' for our running user.

### CODE REVIEW

**Review by maria0990 — APPROVED** [review]

File: instapy/instapy.py [hunk 1]
```diff
@@ -2354,14 +2360,7 @@ def interact_by_users_tagged_posts(self,
...
                                     try:
-                                        checked_img, temp_comments = (
-                                            check_image(self.browser,
-                                                        self.clarifai_api_key,
-                                                        self.clarifai_img_tags,
-                                                        self.clarifai_img_tags_skip,
-                                                        self.logger,
-                                                        self.clarifai_full_match))
-
+                                        checked_img, temp_comments, clarifai_tags = (self.query_clarifai())
```
Comment (timgrossmann) [inline instapy/instapy.py]: What is the reason for the outer parenthesis?

```suggestion
                                        checked_img, temp_comments, clarifai_tags = self.query_clarifai()
```

Comment (sionking) [reply instapy/instapy.py]: just a copy paste ...

Comment (sionking) [reply instapy/instapy.py]: fixed

**Comment by timgrossmann** [comment]
@sionking Perfect, thank you very much!

Could you please also add an entry to the CHANGELOG.md file stating the fix you made, please?

**Comment by CharlesCCC** [comment]
This change make sense to me, because generic call everything as `username` will be confusing.

**Comment by timgrossmann** [comment]
@sionking Merging this now, thank you very much for this update.

**Pull request merged**